server/db_load/db_load.js

```diff
@@ -0,0 +1,54 @@
+const xlsx = require('xlsx')
+const { createClient } = require('@supabase/supabase-js')
+const dotenv = require('dotenv')
+
+dotenv.config({ path: '../.env.production' })
+
+const supabase = createClient(
+  process.env.SUPABASE_URL,
+  process.env.SUPABASE_KEY,
+)
+
+async function loadExcelToSupabase(filePath, tableName) {
+  try {
+    const workbook = xlsx.readFile(filePath)
+    const sheetName = workbook.SheetNames[0]
+    let data = xlsx.utils.sheet_to_json(workbook.Sheets[sheetName])
+
+    data = data.map((item) => ({
+      num: item['Decision Number'],
+      date: item['Decision Date'].split('/').reverse().join('-'),
+      pathname: item['Decision URL'].split('/').slice(-1)[0],
+      title: item['Decision Title'],
+      content: item['Decision Content'],
+      summary: item['Summary'],
+      gov_num: parseInt(item['Government Number'].replace(/\D/g, '')),
+      prime_minister: item['Government Number'].split(', ')?.[1],
```

Comment:
I would add comments for the lines where the string is processed a little - just so that it will be easier later to understand, in any case.